docs/install/methods.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Installation Methods
+---
+
+RKE2 can be installed to a system in a number of ways, 2 of which being the preferred and supported methods. Those methods are tarball and RPM. Both of which are wrappers for the provided install script.
+
+### Tarball
+
+To install RKE2 via install you first need to get the install script. This can be done in a number of ways.
+
+This gets the script and immediately starts the install process.
+
+```bash
+curl -sfL https://get.rke2.io | sh -
+```
+
+This will download the install script and make it executable.
+
+```bash
+curl -sfL https://get.rke2.io --output install.sh
+chmod +x install.sh
+```
+
+#### Installation
+
+The install process defaults to tarball and the latest RKE2 version and no other qualifiers are necessary. However, if you want specify a version, you should set the `INSTALL_RKE2_CHANNEL` environment variable. An example below:
```

Comment:
I think we need to drop the "defaults to tarball" part bc that isnt true on a lot of systems.
```suggestion
The install process defaults to the latest RKE2 version and no other qualifiers are necessary. However, if you want specify a version, you should set the `INSTALL_RKE2_CHANNEL` environment variable. An example below:
```